### PR TITLE
Issue 79- Theming and colors of Application

### PR DESCRIPTION
**Pull request by sachintha-s**
#79
- Dark & light theme added
- Roboto text theme was added as the text theme for the entire app.
- For the light theme, some screens maybe need more styles than now. I think we can do that when the designer finishes the light theme.

### CODE REVIEW

**Comment by gnunicorn** [comment]
> For the light theme, some screens maybe need more styles than now. I think we can do that when the designer finishes the light theme.

Yeah, light theme doesn't have priority at the moment. For the MVP, we won't even allow people to switch the themes yet...

**Review by gnunicorn — CHANGES_REQUESTED** [review]
how about using https://pub.dev/packages/themed or something like it?

File: app/lib/common/store/Colors.dart [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
-  static const Color primaryColor = Color(0xffE9385D);
+  //common colors
+  static const Color primaryColor = Color(0xffEC2758);
   static const Color textFieldColor = Color(0xff171717);
-  static const Color backgroundColor = Color(0xff0A0A0A);
-  static const Color white = Colors.white;
+  static const Color transparent = Colors.transparent;
+  static const Color dividerColor = Color(0xff757575);
+
+  //light theme colors
```
[the remark this answers is not in the document]
Comment (sachintha-s) [reply app/lib/common/store/Colors.dart:10]: Sure, I am trying to use that package.

File: app/lib/common/widget/FaqListItem.dart [hunk 2]
```diff
@@ -16,297 +18,280 @@ class FaqListItem extends StatefulWidget {
...
-            ),
-            // ),
-          ],
-        ));
+                ),
+                elevation: 8,
+                margin: const EdgeInsets.all(20),
+                color: isDarkTheme
+                    ? AppColors.darkFaqCardColor
+                    : AppColors.lightFaqCardColor,
```
[the remark this answers is not in the document]
Comment (sachintha-s) [reply app/lib/common/widget/FaqListItem.dart:284]: Noted. thanks

**Review by gtalha07 — APPROVED** [review]

**Review by gnunicorn — APPROVED** [review]

**Pull request merged**